<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewWeeklyMeetingsDialog</class>
 <widget class="QDialog" name="ViewWeeklyMeetingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lịch hẹn</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Lịch hẹn của bạn</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 14pt; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="gridVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="meetingsLabel">
     <property name="text">
      <string>Danh sách cuộc hẹn trong ngày</string>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="meetingsTree">
     <column>
      <property name="text">
       <string>Thông tin cuộc hẹn</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ViewWeeklyMeetingsDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ViewWeeklyMeetingsDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>